src/views/ResyncRoom.vue

```diff
@@ -22,6 +22,15 @@ const { roomID } = route.params as Record<string, string>
 const sourceInput = ref("")
 const sourceIsURL = computed(() => isURL(sourceInput.value))
 
+watchEffect(() => {
+  // eslint-disable-next-line
```

Comment:
why

Reply:
bc the linter doesnt get the regular expression